src/util/psbt/map/input.rs

```diff
@@ -266,7 +265,7 @@ impl Map for Input {
         }
 
         impl_psbt_get_pair! {
-            rv.push(self.partial_sigs as <PSBT_IN_PARTIAL_SIG, PublicKey>|<Vec<u8>>)
+            rv.push(self.partial_sigs as <PSBT_IN_PARTIAL_SIG, PublicKey>|<PartialSignature>)
```

Comment:
Why here is `PartialSignature` instead of `PartialSig`?

Reply:
I had spotted this bug(not this specific instance) while porting this to rust-elements. The last part of the macro expression for `impl_psbt_get_pair` is not being used. I did not consider it a serious problem but if this can cause this confusion, I think we should clean up the macro in a later PR. At least for this instance, this should be `PartialSig`

Reply:
Created an issue: #754